hoomd/data.py

```diff
@@ -31,28 +31,32 @@
 
 Examples::
 
+    import hoomd
+    system = hoomd.init.create_lattice(unitcell=hoomd.lattice.sc(a=2.0), n=5);
```

Comment:
Please remove trailing semicolons throughout, these are not needed in python examples.